composer.json

```diff
@@ -20,9 +20,13 @@
     },
     "scripts": {
         "check": [
-            "@cs-check"
+            "@cs-check",
+            "@test"
         ],
         "cs-check": "phpcs",
-        "cs-fix": "phpcbf"
+        "cs-fix": "phpcbf",
+        "test": "phpunit --colors=always",
```

Comment:
@froschdesign then this PR should add phpunit to dev dependencies and configuration for it to encourage testing. Otherwise it is neither here or there